emails/2024-09-06.md

```diff
@@ -0,0 +1,27 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Members of the Rust community, affectionately called Rustaceans, come by our Discord occasionally to tell us that Dolt should be written in Rust. It's a straight up Rustacean on Gopher crime. Later in this email, I present Rustaceans an olive branch.
+
+### Dolt Rebase Conflict Resolution
+
+In January, we [release interactive rebase](https://www.dolthub.com/blog/2024-01-03-announcing-dolt-rebase/) even though we're kind of [scared of it](https://www.dolthub.com/blog/2022-12-14-wordcels-guide-to-git/#what-about-git-rebase). This week, we announced you can [now resolve conflicts when rebasing](https://www.dolthub.com/blog/2024-09-05-rebase-conflict-resolution/), a missing feature since launch. 
```

Comment:
```suggestion
In January, we [released interactive rebase](https://www.dolthub.com/blog/2024-01-03-announcing-dolt-rebase/) even though we're kind of [scared of it](https://www.dolthub.com/blog/2022-12-14-wordcels-guide-to-git/#what-about-git-rebase). This week, we announced you can [now resolve conflicts when rebasing](https://www.dolthub.com/blog/2024-09-05-rebase-conflict-resolution/), a missing feature since launch. 
```